ush/parsing_model_configure_FV3.sh

```diff
@@ -31,7 +31,11 @@ local WRITE_GROUP=${WRITE_GROUP:-1}
 local WRTTASK_PER_GROUP=${WRTTASK_PER_GROUP:-24}
 local ITASKS=1
 local OUTPUT_HISTORY=${OUTPUT_HISTORY:-".true."}
-local HISTORY_FILE_ON_NATIVE_GRID=".true."
+if [[ "${DO_JEDIATMVAR:-}" == "YES" ]]; then
+  local HISTORY_FILE_ON_NATIVE_GRID=".true."
+else
+  local HISTORY_FILE_ON_NATIVE_GRID=".false."
```

Comment:
Quick question, when HISTORY_FILE_ON_NATIVE_GRID=".false.", will the softlinks cubed_sphere_grid_atmf???.nc/cubed_sphere_grid_sfcf???.nc  still be created?

Reply:
Good point @junwang-noaa . @aerorahul , logic should be added to forecast_postdet.sh where those soft links are created, depending on whether JEDI is being used

Reply:
Good catch @junwang-noaa 
I'll remove the linking for when this is false.

Reply:
@DavidNew-NOAA 
Not sure how the cubed-sphere restarts got linked in the section where output_grid = Gaussian if-block.

Reply:
@aerorahul Yeah, that was a mistake on my part

Reply:
@aerorahul Well, maybe it was more sloppiness. That ```OUTPUT_GRID``` parameter is somewhat useless, because one cannot write only cubed-sphere histories the way UFS is configured. You either write Gaussian histories or BOTH Gaussian and cubed-sphere histories, depending on how you set ``HISTORY_FILE_ON_NATIVE_GRID```. If I recall from discussion with @CoryMartin-NOAA , that whole block might need refactoring.